### DISCUSSION TITLE
Issues with implementing diffusion!

### DISCUSSION THREAD

**Discussion by rpwh** [RayTracing/raytracing.github.io#1083] [Debugging]
First of all, I think these books are an awesome resource and I'm having a blast working through them and learning new concepts!

I've just ran into this issue working through RayTracingInOneWeekend on section 8.
My sphere larger sphere has these odd lines in it.
![image](https://user-images.githubusercontent.com/22357922/197305114-60feb9ba-bc0e-4272-9427-3f3fdfbae5dc.jpg)

It's probably some off by one error somewhere or something that I can sift through the vode for but was wondering if anyone had experienced this or could point me in the right direction.

This stuff is totally new to me so sorry if this is a silly question.

Thanks!
Rory

**Comment by trevordblack** [comment]
This is a new one that I've not seen. Can you show the code you've written

**Comment by trevordblack** [comment]
This is a new one that I've not seen. Can you show the code you've written

**Reply by rpwh** [reply]
Sure! It's here https://github.com/rpwh/ray-tracing, If I figure out what's causing it I can let you know

**Reply by rpwh** [reply]
I was actually able to fix this by changing t_min in my world.hit() from 0.0 to 0.001. This also drastically improved the runtime too which is nice!

**Reply by rpwh** [reply]
Omg I just started progressing and the very next section addresses this issue.
![image](https://user-images.githubusercontent.com/22357922/197350237-b9d6a7d4-ec55-49ea-bde7-51c58ce898a1.png)

**Reply by calebpapay** [reply]
Hell yes.

I've been banging my head at my wavy ass criss cross shadows for 3 hours
